components/heatmap/HeatMap.vue

```diff
@@ -233,8 +233,8 @@ export default {
     },
     passing_plate_colors: function () {
       return this.well_values[this.display_option].data.map((well) => {
-        const total = well.reduce((a, b) => a + b, 0);
-        const mean = total / well.length;
+        const total = well.slice(-5).reduce((a, b) => a + b, 0);
+        const mean = (total / 5).toFixed(3);
```

Comment:
Should probably make this `5` a const somewhere so we can give it a name and make refactors easier